### DISCUSSION TITLE
UI Parameter Overrides

### DISCUSSION THREAD

**Discussion by andrelsbaires** [argoproj/argo-workflows#9221] [Q&A]
Hello Team,

I'm willing if there is way where Parameters can be override though the UI just as we do with CLI using "-p" 

I've reading the docs and how to's but I'm afraid that I could't find anything about it, perhaps missing something?

Any help on getting this figured out will be more than welcome.

Thank in advance.

**Comment by terrytangyuan** [comment]
Check out this documentation but note that it's not yet released: https://github.com/argoproj/argo-workflows/blob/master/docs/intermediate-inputs.md
